.github/workflows/publish.yml

```diff
@@ -0,0 +1,31 @@
+name: Upload Python Package
+
+on:
+  release:
+    types: published
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
```

Comment:
There is a v2 now that we should consider. Seems like it will only fetch a single commit and is a bit faster. Doesn't really matter for this repository, but I assume we will basically clone this for all repos?

Reply:
Will update!